src/test/java/teammates/test/cases/action/InstructorCourseEnrollSaveActionTest.java

```diff
@@ -240,9 +240,11 @@ public void testExecuteAndPostProcess() throws Exception {
         verifyNoTasksAdded(enrollAction);
         
         //fail to enroll, if exceed the range
-        enrollStringBuilder.append(Const.EOL).append("section" + Const.SIZE_LIMIT_PER_ENROLLMENT + "\tteam" + Const.SIZE_LIMIT_PER_ENROLLMENT
-                                                     + "\tname" + Const.SIZE_LIMIT_PER_ENROLLMENT + "\temail" + Const.SIZE_LIMIT_PER_ENROLLMENT
-                                                     + "@nonexistemail.nonexist");
+        enrollStringBuilder.append(Const.EOL).append(
+                "section" + Const.SIZE_LIMIT_PER_ENROLLMENT + "\tteam" + Const.SIZE_LIMIT_PER_ENROLLMENT
+                        + "\tname" + Const.SIZE_LIMIT_PER_ENROLLMENT + "\temail"
+                        + Const.SIZE_LIMIT_PER_ENROLLMENT
+                        + "@nonexistemail.nonexist");
         submissionParams = new String[]{
```

Comment:
This change is unnecessary if you did not exceed the character limit.
Otherwise, the `+`s should be aligned with `"section"`.